src/dmd/opover.d

```diff
@@ -367,7 +367,7 @@ private Identifier opId_r(Expression e)
  * If type is a class or struct, return the symbol for it,
  * else NULL
  */
-AggregateDeclaration isAggregate(Type t)
+extern (C++) AggregateDeclaration isAggregate(Type t)
```

Comment:
Maybe make this a method of Type? There's already an is Function IIRC.

Reply:
It hurts readability IMO (`isAggregate(e.e2.type)` vs. `e.e2.type.isAggregate()`, used quite a lot). I moved the helper to `dmd.mtype` now though and declare it in the C++ header.